### PR TITLE
feature/api-restructure

### PR DESCRIPTION
**Pull request by jrouzierinverse**
Description
===========
Restructure the rest routes conventions

Impacts
=======
pf::UnifiedApi routing.

NEW Package(s) required
=======================
Lingua::EN::Inflexion
Lingua::EN::Nums2Words

### CODE REVIEW

**Comment by julsemaan** [comment]
Ready for review ?

**Comment by jrouzierinverse** [comment]
Yes

File: addons/packages/packetfence.spec [hunk 1]
```diff
@@ -119,6 +119,8 @@ Requires: perl(Data::Phrasebook), perl(Data::Phrasebook::Loader::YAML)
 Requires: perl(DBI)
 Requires: perl(Rose::DB)
 Requires: perl(Rose::DB::Object)
+Requires: perl(Lingua::EN::Nums2Words) >= 1.16
```
Comment (julsemaan) [inline addons/packages/packetfence.spec:122]: Are these packages built and in the packetfence-devel repo?

**Comment by julsemaan** [comment]
Too a quick look at it, just need confirmation for the packages, then we merge this and then we get onto the final multitenant work for merging in devel

File: addons/packages/packetfence.spec (hunk 1 above)
Comment (jrouzierinverse) [reply addons/packages/packetfence.spec:122]: They are built not but not in packetfence-devel

**Pull request merged**